src/Components/Blazor/Build/testassets/standalone/App.razor

```diff
@@ -0,0 +1,10 @@
+﻿<Router AppAssembly="@typeof(Program).Assembly">
+    <Found Context="routeData">
+        <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
+    </Found>
+    <NotFound>
+        <LayoutView Layout="@typeof(MainLayout)">
+            <p>Sorry, there's nothing at this address.</p>
+        </LayoutView>
+    </NotFound>
+</Router>
```

Comment:
Commenting on the whole of `src/Components/Blazor/Build/testassets/standalone` here:

This is quite a lot of arbitrary detail about the project template getting duplicated here, and yet another copy of it we have to update everytime the template changes otherwise it's confusing why some things are in sync and some aren't.

Instead of all this, would it make sense to reduce the entire app here just to the `index.html`, `Startup.cs`, `Program.cs` and `App.razor` files and have it simply emit "Hello world"?

Reply:
We will also soon have E2E tests for the blazorwasm template, so I think that will give us coverage of this typical app scenario anyway.

Reply:
Done